engine/platform/nswitch/sys_nswitch.c

```diff
@@ -0,0 +1,140 @@
+/*
+switch.c - switch backend
+Copyright (C) 2021-2023 fgsfds
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+*/
+
+#include "platform/platform.h"
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <sys/stat.h>
+#include <switch.h>
+#include <solder.h>
+#include <SDL.h>
+
+static int nxlink_sock = -1;
+
+/* HACKHACK: force-export stuff required by the dynamic libs */
+
+// libunwind stuff for C++ exceptions, required by filesystem_stdio
```

Comment:
C++ exceptions in filesystem_stdio weren't supposed to be enabled... 

But anyway, let's keep _Unwind symbols anyway for mod compatibility.

Reply:
Turns out the build system was linking libstdc++ to everything by mistake. Fixed in 69607d78.